apps/system/iperf/Makefile

```diff
@@ -100,7 +97,7 @@ endif
 
 ifeq ($(CONFIG_BUILTIN_APPS)$(CONFIG_SYSTEM_IPERF),yy)
 $(BUILTIN_REGISTRY)$(DELIM)$(FUNCNAME).bdat: $(DEPCONFIG) Makefile
-	$(Q) $(call REGISTER,$(APPNAME),$(FUNCNAME),$(THREADEXEC),$(IPERF_PRIORITY),$(IPERF_STACKSIZE))
+	$(Q) $(call REGISTER,$(APPNAME),$(FUNCNAME),$(THREADEXEC),$(CONFIG_IPERF_PRIORITY),$(CONFIG_IPERF_STACKSIZE))
```

Comment:
We recommend to use this sentence like below:
`$(Q) $(call REGISTER,$(APPNAME),$(FUNCNAME),$(THREADEXEC),$(PRIORITY),$(STACKSIZE))`
So please declare and assign `PRIORITY` and `STACKSIZE` in the above

Reply:
Modified. Thanks.